tests/publiccloud/ipa.pm

```diff
@@ -41,21 +42,44 @@ sub create_secgroup {
     return find_secgroup($name);
 }
 
+sub create_ssh_key {
+    my ($prefix, $out_file) = @_;
+
+    for my $i (0 .. 9) {
+        my $key_name = $prefix . "_" . $i;
+        my $cmd      = "aws ec2 create-key-pair --key-name '" . $key_name . "' --query 'KeyMaterial' --output text > " . $out_file;
+        if (script_run($cmd) == 0) {
```

Comment:
Why not just use `assert_script_run` when you would die in L55 anyway?

Reply:
I use a different name on the next try, so It could work. And die only after ten retries.